<ui version="4.0" >
 <class>ODocMainWindow</class>
 <widget class="QMainWindow" name="ODocMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ODoc</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../Art/Avatars/Avatartux.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTreeView" name="Tree" >
      <property name="baseSize" >
       <size>
        <width>144</width>
        <height>512</height>
       </size>
      </property>
      <property name="selectionBehavior" >
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="animated" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Trab</string>
       </attribute>
       <widget class="QTextBrowser" name="Viewer" >
        <property name="geometry" >
         <rect>
          <x>9</x>
          <y>9</y>
          <width>363</width>
          <height>471</height>
         </rect>
        </property>
        <property name="baseSize" >
         <size>
          <width>432</width>
          <height>512</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow" />
    <addaction name="actionAddTab" />
    <addaction name="actionClose_Tab" />
    <addaction name="separator" />
    <addaction name="actionPrint" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy" />
    <addaction name="action_Find" />
   </widget>
   <widget class="QMenu" name="menu_Go" >
    <property name="title" >
     <string>&amp;Go</string>
    </property>
    <addaction name="action_Previous" />
    <addaction name="action_Next" />
    <addaction name="action_Home" />
    <addaction name="separator" />
    <addaction name="action_Previous_Tab" />
    <addaction name="action_Next_Tab" />
   </widget>
   <widget class="QMenu" name="menu_Bokmarks" >
    <property name="title" >
     <string>&amp;Bokmarks</string>
    </property>
    <addaction name="actionNewBookmark" />
   </widget>
   <addaction name="menuOpenFile" />
   <addaction name="menuEdit" />
   <addaction name="menu_Go" />
   <addaction name="menu_Bokmarks" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
  </widget>
  <action name="actionNewWindow" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;New Window</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAddTab" >
   <property name="text" >
    <string>Add &amp;Tab</string>
   </property>
  </action>
  <action name="actionClose_Tab" >
   <property name="text" >
    <string>&amp;Close Tab</string>
   </property>
  </action>
  <action name="actionPrint" >
   <property name="text" >
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Copy" >
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Find" >
   <property name="text" >
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="action_Previous" >
   <property name="text" >
    <string>&amp;Previous</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="action_Next" >
   <property name="text" >
    <string>&amp;Next</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="action_Home" >
   <property name="text" >
    <string>&amp;Home</string>
   </property>
   <property name="shortcut" >
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="action_Previous_Tab" >
   <property name="text" >
    <string>&amp;Previous Tab</string>
   </property>
  </action>
  <action name="action_Next_Tab" >
   <property name="text" >
    <string>&amp;Next Tab</string>
   </property>
  </action>
  <action name="actionNewBookmark" >
   <property name="text" >
    <string>&amp;New Bookmark</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>ODocMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
